Utility/Connect-AzureVM.ps1

```diff
@@ -1,20 +1,20 @@
 <#PSScriptInfo
```

Comment:
Should this be a seperate runbook from the old Connect-AzureVM? So that ASM users can still use that one?